modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/DDMServiceUpgrade.java

```diff
@@ -230,8 +230,10 @@ public void register(Registry registry) {
 
 		registry.register("2.0.7", "2.0.8", new DummyUpgradeStep());
 
+		registry.register("2.0.8", "2.0.9", new DummyUpgradeStep());
+
 		registry.register(
-			"2.0.8", "3.0.0",
+			"2.0.9", "3.0.0",
```

Comment:
@rodrigopaulino, why was it necessary?

Reply:
This is necessary every time an Upgrade needs to be backported. This reserves the version number that's going to be used on 7.1.x.

Reply:
Practically, if someone is trying to upgrade from 7.1 to the latest, Liferay will be able to pick it up from their schema version onwards.